pulp_smash/tests/pulp3/pulpcore/api_v3/test_tasks.py

```diff
@@ -0,0 +1,91 @@
+# coding=utf-8
+"""Test that operations can be performed over tasks."""
+import unittest
+
+from requests import HTTPError
+
+from pulp_smash import api, config, selectors, utils
+from pulp_smash.tests.pulp3.constants import REPO_PATH, TASKS_PATH
+from pulp_smash.tests.pulp3.pulpcore.utils import gen_repo
+from pulp_smash.tests.pulp3.utils import get_auth
+from pulp_smash.tests.pulp3.pulpcore.utils import set_up_module as setUpModule  # pylint:disable=unused-import
+
+_DYNAMIC_TASKS_ATTRS = ('finished_at',)
+"""Task attributes that are dynamically set by Pulp, not set by a user."""
+
+_P3_TASK_END_STATES = ('canceled', 'completed', 'failed', 'skipped',)
```

Comment:
This can be imported from `pulp_smash.api`. I know it's a Bad Idea to import private variables, but given that this module specifically tests the tasking system, the benefit of having a single variable seems to outweigh the improper-ness of accessing private data. The variable could also be made public, if you prefer.

Reply:
@Ichimonji10, what about move to Pulp3 constants?